test/src/module/config/parseTest.c

```diff
@@ -816,6 +816,21 @@ testRun(void)
         TEST_RESULT_STR_Z(cfgCommandRoleStr(cfgCmdRoleRemote), "remote", "    remote role name");
         TEST_RESULT_INT(logLevelStdOut, logLevelError, "console logging is error");
         TEST_RESULT_INT(logLevelStdErr, logLevelError, "stderr logging is error");
+
+        argList = strLstNew();
+        strLstAdd(argList, strNew("pgbackrest"));
+        hrnCfgArgRawZ(argList, cfgOptPgPath, "/path/to");
+        strLstAdd(argList, strNew("--stanza=db"));
+        strLstAdd(argList, strNew("--log-level-stderr=info"));
```

Comment:
I tried this same test with archive-get because I did not see a test where the default proves it is reset. In any event, it does reset the stderr to a warning vs the requested info - and although I can't find it - I assume there must be somewhere in the config that we restore the logging to the levels requested after we're done parsing the config?

Reply:
Per discussion with David, the settings for log-levels requested by the user, will get reapplied after the parsing, by calling cfgLoadLogSetting(). And the tests that prove the logging is set to warning then reset are partly here but mostly in the load tests.